params/config.go

```diff
@@ -339,9 +339,10 @@ type ChainConfig struct {
 	// Astria Specific Configuration
 	AstriaOverrideGenesisExtraData       bool          `json:"astriaOverrideGenesisExtraData,omitempty"`
 	AstriaExtraDataOverride              hexutil.Bytes `json:"astriaExtraDataOverride,omitempty"`
-	AstriaSequencerInitialHeight         *big.Int      `json:"astriaDequencerInitialHeight,omitempty"`
-	AstriaDataAvailabilityInitialHeight  *big.Int      `json:"astriaDataAvailabilityInitialHeight,omitempty"`
-	AstriaDataAvailabilityHeightVariance *big.Int      `json:"astriaDataAvailabilityHeightVariance,omitempty"`
+	AstriaRollupName                     string        `json:"astriaRollupName,omitempty"`
+	AstriaSequencerInitialHeight         uint32        `json:"astriaDequencerInitialHeight,omitempty"`
```

Comment:
Two change suggestions here:
1. this was named "dequencer" for some reason
2. I don't think `omitempty` carries the right semantics - the marshaller will populate this with `0` if missing, which cannot be the case (same for celestia):
```suggestion
	AstriaSequencerInitialHeight         uint32        `json:"astriaSequencerInitialHeight"`
```